src/main/java/frc/robot/devices/ColorSensor.java

```diff
@@ -19,6 +20,20 @@
     private final Color blueTarget;
     private final Color redTarget;
     private final Color noTarget;
+    private static double measuredProximity;
+    private static Color measuredColor;
```

Comment:
Why are these static? They should be instance variables. Just because it is multithreaded doesn't mean that it should be static.

Reply:
I am not sure why I did that, but it is fixed now.